web/ui/react-app/src/Panel.tsx

```diff
@@ -36,7 +37,11 @@ interface PanelState {
   } | null;
   loading: boolean;
   error: string | null;
-  stats: null; // TODO: Stats.
+  stats: {
```

Comment:
I would suggest you create interface PaneStats or something like that and to use it here and in QueryStats component as well

Reply:
Did some renaming, and shared it from QueryStats.